shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -668,32 +672,64 @@ public long configureForTextureLayerComposition(
     return textureId;
   }
 
+  /**
+   * Translates an original touch event to have the same locations as the ones that Flutter
+   * calculates (because original + flutter's - original = flutter's).
+   *
+   * @param originalEvent The saved original input event.
+   * @param pointerCoords The coordinates that Flutter thinks the touch is happening at.
+   */
+  private void translateNonVirtualDisplayMotionEvent(
```

Comment:
I don't think you should include "NonVirtualDisplay" in the name, because this function on its own simply "translates a motion event" the fact that we only call it in the non-virtual-display-case isn't important to the name of this function (as it could be used by others for other reasons).